components/net/image_cache.rs

```diff
@@ -224,7 +218,7 @@ impl ImageBytes {
                 ImageBytes::InProgress(ref mut bytes) => bytes,
                 ImageBytes::Complete(_) => panic!("attempted modification of complete image bytes"),
             };
-            mem::replace(own_bytes, vec![])
+            std::mem::take(own_bytes)
```

Comment:
Either `mem::take` or stop importing `std::mem`, but this is causing a compile warning (#31607).

Reply:
Sorry about that, I'll fix it right away.

Reply:
Should be fixed with #31608